packages/blockchain-wallet-v4/src/network/api/btc/BtcAPI.d.ts

```diff
@@ -0,0 +1,10 @@
+export type BtcAPI = {
+  getBtcTicker: () => any
```

Comment:
Overall, the main benefits here would be to produce the data models interfaces that we expect coming from our api endpoints to get full benefits of TS. I know its just a small POC, but to think about it for later !

Reply:
Yes, 100% agree, the goal would be to generate these automatically and do the actual work in the respective index.ts files.